sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_patch.py

```diff
@@ -18,3 +22,82 @@ def patch_sdk():
     you can't accomplish using the techniques described in
     https://aka.ms/azsdk/python/dpcodegen/python/customize
     """
+
+
+class ContentSafetyClientOperationsMixin(ContentSafetyClientOperationsMixinGenerated):
+    @overload
+    def analyze_text(
+        self, body: str, *, content_type: str = "application/json", **kwargs: Any
+    ) -> _models.AnalyzeTextResult:
+        """Analyze Text.
+
+        A synchronous API for the analysis of potentially harmful text content. Currently, it supports
+        four categories: Hate, SelfHarm, Sexual, and Violence.
+
+        parameter body: The text analysis request. Required.
+        type body: string
+        :keyword content_type: Body Parameter content-type. Content type parameter for JSON body.
+         Default value is "application/json".
+        parameter content_type: str
+        :keyword bool stream: Whether to stream the response of this operation. Defaults to False. You
+         will have to context manage the returned stream.
+        :return: AnalyzeTextResult. The AnalyzeTextResult is compatible with MutableMapping
+        :rtype: ~azure.ai.contentsafety.models.AnalyzeTextResult
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        ...
+
+    def analyze_text(self, body: str, **kwargs: Any) -> _models.AnalyzeTextResult:
+        options = _models.AnalyzeTextOptions(text=body)
+        return super().analyze_text(body=options, **kwargs)
```

Comment:
If the body only receive one type `str`, I think there is no need to add `@overload` function anymore.

Reply:
We have the analyze_text overloads take `_models.AnalyzeTextOptions(...)` as paramter. please check [_operations.py](https://github.com/JieZhou000/azure-sdk-for-python/blob/zhojie/py-0/sdk/contentsafety/azure-ai-contentsafety/azure/ai/contentsafety/_operations/_operations.py) line 305